src/FsAutoComplete.Core/Lexer.fs

```diff
@@ -240,3 +240,14 @@ module Lexer =
                     | head :: tail -> tail |> List.rev, head
                     | [] -> [], ""
         | _ -> [], ""
+
+    let findClosestIdent col lineStr =
```

Comment:
this is the core of the new signaturehelp features: walking back along the long idents from a given position to find the whole span+parts of the long ident.